lib/src/dsl_util.rs

```diff
@@ -14,6 +14,7 @@
 
 //! Domain-specific language helpers.
 
+use core::ascii;
```

Comment:
nit: just import from `std::`?

Reply:
Done. (VS code does the imports and I usually don't check.)